docs/content/direct/common-setup-kind.md

```diff
@@ -0,0 +1,100 @@
+# KubeStellar Core Sample Setup -- Kind with OCM
```

Comment:
Why is "with OCM" here? As far as I know, we do not have "without OCM" option.

Reply:
It was for parallelism in the list of sample setups, going back to a slack conversation a couple months ago.